external/common/share/capnproto/README.md

```diff
@@ -1,2 +1,6 @@
-capnproto-c++-0.5.3.tar.gz: from https://capnproto.org/capnproto-c++-0.5.3.tar.gz
-capnproto-c++-win32-0.5.3.zip: from https://capnproto.org/capnproto-c++-win32-0.5.3.zip
+capnproto-c++-0.5.3.tar.gz: from https://capnproto.org/capnproto-c++-0.5.3.1.tar.gz
+capnproto-c++-win32-0.5.3.zip: from https://capnproto.org/capnproto-c++-win32-0.5.3.1.zip
+
+When pycapnp (in requirements.txt) is updated, please check with which version of capnproto it ships and update the files also here
+and in external/Capnproto.cmake ! 
```

Comment:
>  why does it matter if pycapnp's capnproto version matches capnproto version here? 

Sorry, I'm back from holiday, so I may have lost context in here, but the issue with this started from : 
https://github.com/capnproto/pycapnp/issues/152#issuecomment-303255346 

So I think the incompatibility was the compiler picked up the (older) capnp version shipped in nupic.core, while newer (by only a x.y.z.1) version was required. I would ask otherwise, do we need this  custom installed capnp binary, could we just use the one shipped as dependency of pycapnp? 

TL:DR: clang 4.x breaks when these two capnp versions are not in sync, and it isn't documented in nupic.core readmes